frontend/src/components/OdhDocListItem.tsx

```diff
@@ -93,7 +99,7 @@ const OdhDocListItem: React.FC<OdhDocCardProps> = ({ odhDoc, favorite, updateFav
       </div>
       <div className="odh-list-item__doc-text">
         {odhDoc.spec.appDisplayName ? (
-          <div id={odhDoc.spec.appDisplayName} className="odh-list-item__doc-title">
+          <div id={itemId} className="odh-list-item__doc-title">
```

Comment:
Why did we do this change? Was there an issue with A11y?

Reply:
As discussed, all ids in this screen should be unique (per aXe devtools)!